Makefile

```diff
@@ -101,9 +104,8 @@ codespell:
 
 lint: pyspec
 	. venv/bin/activate; cd $(PY_SPEC_DIR); \
-	flake8  --ignore=E252,W504,W503 --max-line-length=120 ./eth2spec \
-	&& cd ./eth2spec && mypy --follow-imports=silent --warn-unused-ignores --ignore-missing-imports --check-untyped-defs --disallow-incomplete-defs --disallow-untyped-defs -p phase0 \
-	&& mypy --follow-imports=silent --warn-unused-ignores --ignore-missing-imports --check-untyped-defs --disallow-incomplete-defs --disallow-untyped-defs -p phase1;
```

Comment:
Is `--check-untyped-defs` redundant and omitted from the config intentionally?

Reply:
Yes, we already have `disallow_untyped_defs = True` (`--disallow-untyped-defs`), which is more severe than `--check-untyped-defs`.